centml/templates/notebook-service.yaml

```diff
@@ -1,21 +1,28 @@
 apiVersion: v1
 kind: Service
 metadata:
-  {{- if .Values.service.annotations }}
+  type: {{ .Values.service.type }}
   annotations:
-    {{- toYaml .Values.service.annotations | nindent 4 }}
-  {{- end }}
+    {{- if .Values.service.publicIP }}
+    metallb.universe.tf/address-pool: {{ .Values.service.annotions.addressPool }}
+    metallb.universe.tf/allow-shared-ip: {{ .Values.service.annotions.allowSharedIp }}
+    external-dns.alpha.kubernetes.io/hostname: {{ .Values.service.annotions.externalDNSHostName }}
```

Comment:
Please just template this from the release name instead of asking the user to enter the full dns. i.e. {{.Release.Name}}.{{.Release.Namespace}}.coreweave.cloud

Reply:
Changed it according to `{{.Release.Name}}.{{.Release.Namespace}}.coreweave.cloud`